dot/sync/chain_sync_test.go

```diff
@@ -1132,15 +1132,15 @@ func Test_chainSync_logSyncSpeed(t *testing.T) {
 				blockStateBuilder: func(ctrl *gomock.Controller) BlockState {
 					mockBlockState := NewMockBlockState(ctrl)
 					mockBlockState.EXPECT().BestBlockHeader().Return(&types.Header{}, nil).AnyTimes()
-					mockBlockState.EXPECT().GetHighestFinalisedHeader().Return(&types.Header{}, nil)
+					mockBlockState.EXPECT().GetHighestFinalisedHeader().Return(&types.Header{}, nil).AnyTimes()
 					return mockBlockState
 				},
 				networkBuilder: func(ctrl *gomock.Controller, done chan struct{}) Network {
 					mockNetwork := NewMockNetwork(ctrl)
 					mockNetwork.EXPECT().Peers().DoAndReturn(func() error {
 						close(done)
 						return nil
-					})
+					}).AnyTimes()
```

Comment:
Please revert this file, https://github.com/ChainSafe/gossamer/pull/2610 should fix it 😉